### PR TITLE
Release: 2.4.1

### PR DESCRIPTION
**Pull request by aboedo**
## 2.4.1

- Bump`purchases-hybrid-common` to `2.0.0` [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/2.0.0)
- Bump `purchases-ios` to `3.14.1` ([Changelog here](https://github.com/RevenueCat/purchases-ios/releases/3.14.1))
- Bump `purchases-android` to `4.6.0` ([Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.6.0))

### CODE REVIEW

File: src/ios/CDVPurchasesPlugin.m [hunk 1]
```diff
@@ -29,7 +29,8 @@ - (void)setupPurchases:(CDVInvokedUrlCommand *)command {
                         observerMode:observerMode
                userDefaultsSuiteName:userDefaultsSuiteName
                       platformFlavor:self.platformFlavor
-               platformFlavorVersion:self.platformFlavorVersion];
+               platformFlavorVersion:self.platformFlavorVersion
+                   dangerousSettings:nil];
```
Comment (aboedo) [inline src/ios/CDVPurchasesPlugin.m:33]: this is the breaking change from hybridcommon 2.0

File: VERSIONS.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 | ------- | ----------- | --------------- | -------------------- |
+| 2.4.0   | 3.14.1      | 4.6.0           | 2.0.0                |
```
Comment (tonidero) [inline VERSIONS.md:3]: I think this needs updating. I did a quick check and I believe it should be this:
```suggestion
| 2.4.1   | 3.14.0      | 4.6.1           | 2.0.0                |
```

File: src/ios/CDVPurchasesPlugin.m (hunk 1 above)
Comment (tonidero) [reply src/ios/CDVPurchasesPlugin.m:33]: Hmm looks like this was done in iOS only. I believe this is correct but just to confirm, android didn't have that change then right?

**Review by tonidero — APPROVED** [review]
LGTM!

File: src/ios/CDVPurchasesPlugin.m (hunk 1 above)
Comment (aboedo) [reply src/ios/CDVPurchasesPlugin.m:33]: It's iOS only. I don't quite recall _why_ 😅 

https://github.com/RevenueCat/purchases-hybrid-common/releases/2.0.0

https://github.com/RevenueCat/purchases-hybrid-common/blob/2.0.0/android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt#L320

Comment (aboedo) [reply src/ios/CDVPurchasesPlugin.m:33]: it was added in 3.0.0
https://github.com/RevenueCat/purchases-hybrid-common/compare/2.0.0...3.0.0#diff-77d224542d5cfe0397ffb1d5cd7af37a219d8587772ed4d63d777ab3d0f0bc91R294

we had to update to the next major for iOS and Android as separate tasks, I think that's why we did this

File: VERSIONS.md (hunk 2 above)
Comment (aboedo) [reply VERSIONS.md]: updated. I also updated to 2.0.1, I missed that version before

**Comment by aboedo** [comment]
Published as hotfix tag, closing this PR.

**Pull request closed without merging**